page_objects/examples/weather_shopper_mobile_app/weather_shopper_cart_page.py

```diff
@@ -0,0 +1,70 @@
+"""
+Page objects for cart page in Weathershopper application.
+"""
+# pylint: disable = W0212,E0401
+import re
+import conf.locators_conf as locators
+from utils.Wrapit import Wrapit
+from core_helpers.mobile_app_helper import Mobile_App_Helper
+
+class WeatherShopperCartPage(Mobile_App_Helper):
+    "Page object for the cart page in Weathershopper application."
+
+    @Wrapit._exceptionHandler
+    @Wrapit._screenshot
+    def get_cart_total(self):
+        "This method is to get the total price in the cart."
+
+        cart_total = self.get_text(locators.total_amount)
+        # Extracting the numeric part using regular expression
+        match = re.search(r'\d+\.\d+', cart_total.decode())
+        total_amount = float(match.group()) if match else None
+
+        if total_amount is None:
+            self.write("Total amount is None", level='debug')
+        else:
+            self.write(f"Total amount is {total_amount}", level='debug')
+        return total_amount
+
+    @Wrapit._exceptionHandler
+    @Wrapit._screenshot
+    def verify_total(self, cart_total, cart_item_1=0, cart_item_2=0):
```

Comment:
Good to update this method to support the multiple cart items.  (cart_total and list of cart items)

Reply:
Updated the method to use list